frames/bodypack.go

```diff
@@ -0,0 +1,975 @@
+package frames
+
+var Forrest = DefaultFrameType(forrestFrames)
+
+var forrestFrames = []string{
+	                                                                                
+                                                                                
+                                                                                
+                                                                                
+                                                                                
+                                                                                
+                                                                                
+                                       ...                                      

```

Comment:
This doesn't compile, you'll need to surround each frame in backticks (<code>`</code>).

Reply:
E.g.
```golang
var myAnimationFrames = []string{
  `Frame1`,
  `Frame2`,
  `Frame3`,
}
```